### PR TITLE
Migrated to Grails-3.0.2

### PR DESCRIPTION
**Pull request by mansi90**

### CODE REVIEW

**Comment by aruizca** [comment]
Hi @mansi90 
Thanks for your contribution. I will review the PR and release. TBH I have not released a Grails 3 plugin yet, so I have to get familiarized with it first.

File: test/unit/org/grails/plugins/google/visualization/GoogleVisualizationTagLibTests.groovy [hunk 1]
```diff
@@ -1,88 +0,0 @@
...
-
-/**
- * Google visualization taglib tests
- *
- * @author <a href='mailto:benjamin.muschko@gmail.com'>Benjamin Muschko</a>
- */
-@TestMixin(GroovyPageUnitTestMixin)
-@TestFor(GoogleVisualizationTagLib)
-class GoogleVisualizationTagLibTests {
-    void testPieChart() {
```
Comment (aruizca) [inline test/unit/org/grails/plugins/google/visualization/GoogleVisualizationTagLibTests.groovy]: Why was this test removed?

Comment (meudaypratap) [inline test/unit/org/grails/plugins/google/visualization/GoogleVisualizationTagLibTests.groovy]: Test cases are not removed they are migrated to spock test cases

Comment (aruizca) [inline test/unit/org/grails/plugins/google/visualization/GoogleVisualizationTagLibTests.groovy]: Gotcha! :-) :+1:

**Pull request merged**